pkgs/top-level/all-packages.nix

```diff
@@ -16559,6 +16559,8 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) ApplicationServices;
   });
 
+  imapnotify = nodePackages.imapnotify;
```

Comment:
Can you do inherit here instead? Eventually I want to merge all of these & that makes it a little cleaner.